.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,116 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
This file should be removed from PR + .idea folder can be added to .gitignore - we don't need IDE specific files in repo